src/models/Event.js

```diff
@@ -20,32 +20,35 @@ const eventSchema = new Schema({
         minlength: [5, "Description must be minimum 5 characters long!"],
         maxlength: [200, "Description must be maximum 200 characters long!"],
     },
-    dates: [{
-        startDate: { type: Date, required: true },
-        startTime: {
-            type: String,
-            required: true,
-            minlength: [5, "Hour must be in format: 'hh:mm'!"],
-            maxlength: [5, "Hour must be in format: 'hh:mm'!"],
-        },
-        endTime: {
-            type: String,
-            required: true,
-            minlength: [5, "Hour must be in format: 'hh:mm'!"],
-            maxlength: [5, "Hour must be in format: 'hh:mm'!"],
-        }
-    }, { // Additional date and time (if needed)
-        date: { type: Date },
-        startTime: { type: String },
-        endTime: { type: String }
-    }],
+    dates: {
+        type: [{
+            startDate: { type: Date, required: true },
+            startTime: {
+                type: String,
+                required: true,
+                minlength: [5, "Hour must be in format: 'hh:mm'!"],
+                maxlength: [5, "Hour must be in format: 'hh:mm'!"],
+            },
+            endTime: {
+                type: String,
+                required: true,
+                minlength: [5, "Hour must be in format: 'hh:mm'!"],
+                maxlength: [5, "Hour must be in format: 'hh:mm'!"],
+            }
+        }, { // Additional date and time (if needed)
```

Comment:
Is this needed? Isn’t the upper date an array of N date objects?

Reply:
![image](https://github.com/Getsov/Motorsport-Events-Server/assets/50380455/44f6d2f0-a3e2-43c0-81f0-7d38910fe126)

Reply:
I did't work with date objects Much, but if neededd just say it! Instead i put a regexp validation on inputs for models for now :) and has been tested!